scripts/gulpfiles/build_tasks.js

```diff
@@ -116,7 +116,8 @@ function compile(compilerOptions, opt_verbose, opt_warnings_as_error,
   const options = {};
   options.compilation_level = 'SIMPLE_OPTIMIZATIONS';
   options.warning_level = opt_verbose ? 'VERBOSE' : 'DEFAULT';
-  options.language_in = 'ECMASCRIPT5_STRICT';
+  // Use stable, regardless of what was passed in.
+  options.language_in = 'STABLE';
   options.language_out = 'ECMASCRIPT5_STRICT';
   options.rewrite_polyfills = false;
```

Comment:
This line (`options.rewrite_polyfills = false`) is also relevant to this change, since it overrides a default and we may want those polyfills.